package.json

```diff
@@ -9,15 +9,15 @@
   },
   "types": "index.d.ts",
   "scripts": {
-    "test": "mocha 'test/**/*.spec.js'",
-    "test-cov": "nyc mocha 'test/**/*.spec.js'",
+    "test": "mocha \"test/**/*.spec.js\"",
```

Comment:
Is this also to make Windows happy?

Reply:
Yeah, some terminals seem to blow up with single quotes. But that may be me or my VM? In any case, this works 😛